common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1296,6 +1296,9 @@ def get_submission_metadata(self, answers, correct_map):
                 'correct': is_correct,
                 'variant': variant,
             }
+            # Add group_label in event data only if the responsetype contains multiple inputtypes
+            if answer_input.response_data.get('group_label'):
+                input_metadata[input_id]['group_label'] = answer_input.response_data.get('group_label')
```

Comment:
Can we move this with rest of the context, i think it should not be conditional, meta should have `group_label` key all the time.